src/start_fact.py

```diff
@@ -56,13 +53,17 @@ def _evaluate_optional_args(args: argparse.Namespace):
 
 def _start_component(component: str, args: argparse.Namespace) -> Popen | None:
     script_path = Path(get_src_dir()) / f'../start_fact_{component}'
-    if not script_path.exists():
-        logging.debug(f'{component} not installed')
-        return None
-    logging.info(f'starting {component}')
```

Comment:
Why did you remove this?

Reply:
I don't see any benefit in logging that FACT started. Isn't that kinda obvious?
If we printed some information about FACT (e.g. version) like postgres and redis do I'd see a reason to print this.
But I will revert this change if you disagree.

Redis: ` # Redis version=6.0.16, bits=64, commit=00000000, modified=0, pid=677410, just started`

postgresql: `starting PostgreSQL 14.6 (Ubuntu 14.6-1.pgdg22.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0, 64-bit`

Reply:
Since FACT heavily relies on multiprocessing, I find it somewhat reassuring to see that every component was started successfully. I would say that it makes sense for logging level "info". I can also understand why you think that it does not actually convey any useful information, though. IMHO we should at least move it to logging.debug instead of removing it. Has @dorpvom an opinion on this?

I agree that we should log some version info. That could e.g. be useful for error logs in the new issue template :sweat_smile: